dashbuilder-website-docs/src/main/asciidoc/guides/yaml_guide.adoc

```diff
@@ -191,6 +191,14 @@ pages:
 - name: DataSet sample
 ----
 
+Datasets contains fields as follows:-
```

Comment:
There's still a leftover hyphen `-`. I would also rephrase to `Datasets can have the following fields:`

Thanks!